packages/ckeditor5-dev-release-tools/lib/tasks/commitandtag.js

```diff
@@ -54,6 +54,6 @@ export default async function commitAndTag( {
 	} else if ( !tagForVersion ) {
 		// Commit and create a tag if it does not exist yet. It might happen when a release job is restarted.
 		await makeCommit();
-		await git.addTag( `v${ version }` );
+		await git.addAnnotatedTag( `v${ version }`, `Release: v${ version }.` );
```

Comment:
Why `addAnnotatedTag`?

Reply:
Ok, got it.

![image](https://github.com/user-attachments/assets/f692d163-b47e-403c-8273-a98ca0c95e05)

Reply:
because `addTag` does not accept two arguments.